src/app/Authentication/registration/registration.component.css

```diff
@@ -0,0 +1,47 @@
+.registration-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+}
+
+form {
+    width: 300px;
+    margin: 20px;
+    padding: 20px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
```

Comment:
noted apu

Reply:
as this will take changes a lot of changes, im avoiding it for now. 
But this was a good suggestion. i will try to follow it from now. Thank you apu

Reply:
This is not the proper way to resolve any reviews. 
`As this will take a long time` depends on many factors such as client emergency or hot fix. 
This project was given to you to learn the best practices, understand, and apply them appropriately doesn't matter how long it takes. Also looking at the files, I don't think it would take that much time either. 
Please properly resolve the review.

Reply:
Okay bhai.. working on it